pulsar-client/src/test/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImplTest.java

```diff
@@ -62,4 +73,45 @@ public void testGetStats() throws Exception {
         impl.getStats();
     }
 
+    @Test
+    public void multiTopicsInDifferentNameSpace() throws PulsarClientException {
+        List<String> topics = new ArrayList<>();
+        topics.add("persistent://public/default/MultiTopics1");
+        topics.add("persistent://public/test-multi/MultiTopics2");
+        topics.add("persistent://public/test-multi/MultiTopics3");
+        ClientConfigurationData conf = new ClientConfigurationData();
+        conf.setServiceUrl("pulsar://localhost:6650");
```

Comment:
@huangdx0726 Seems this is related to your error. since this test `MultiTopicsConsumerImplTest` not start pulsar service at all.  
Please move this testcast into `TopicsConsumerImplTest`, which contains all needed setups.

Reply:
I have fixed as persd your comment, PTAL.